Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -608,8 +608,11 @@ UpdateFspConfig (
         Fspmcfg->PcieClkReqGpioMux[9] = 0x796e9000;
         Fspmcfg->SkipCpuReplacementCheck = 0x0;
         break;
-        case PLATFORM_ID_ADL_PS_DDR5_CRB:
-          Fspmcfg->DdiPortAConfig = 0x1;
+      case PLATFORM_ID_ADL_PS_DDR5_CRB:
+        Fspmcfg->DdiPortAConfig = 0x0;
+        Fspmcfg->PcieClkReqGpioMux[9] = 0x796e9000;
+        Fspmcfg->SkipCpuReplacementCheck = 0x0;
+        Fspmcfg->FirstDimmBitMaskEcc = 0x0;
```

Comment:
Please add FirstDimmBitMaskEcc directly to the dlt file. This is not added as part of the base configuration yaml file

Reply:
@sindhuragrandhi, will work on it.

Reply:
@sindhuragrandhi, find some merge conflict, will close this one and open new PR.